src/accordion/accordion.ts

```diff
@@ -102,6 +110,13 @@ export class NgbAccordion implements AfterContentChecked {
   @Input('closeOthers') closeOtherPanels: boolean = false;
 
   /**
+   *  Accordion-Panel type (CSS class). Bootstrap 4 recognizes the following types: "success", "info", "warning" and
+   * "danger".
+   */
+  @Input() type = 'default';
```

Comment:
If I'm not mistaken `card-default` doesn't exist in BS4 (at least not in alpha.2). As such we shouldn't be adding it. Instead we should add any type-related class when type is not specified.